charmtools/bundles.py

```diff
@@ -40,6 +41,13 @@ def proof(self, bundle):
                 'for the supported format.')
             return
 
+        # Pushing a bundle to the charm store (Juju 2+) requires a bundle.yaml.
+        ver = get_juju_version()
+        if ver and ver >= 2.0:
```

Comment:
Is bundle.json also deprecated, i.e., bundle.yaml is the only supported
file name going forward?
On Tue, Apr 19, 2016 at 12:52 AM Marco Ceppi <notifications@github.com>
wrote:

> In charmtools/bundles.py
> <https://github.com/juju/charm-tools/pull/185#discussion_r60173042>:
>
> > @@ -40,6 +41,13 @@ def proof(self, bundle):
> >                  'for the supported format.')
> >              return
> >
> > +        # Pushing a bundle to the charm store (Juju 2+) requires a bundle.yaml.
> > +        ver = get_juju_version()
> > +        if ver and ver >= 2.0:
>
> This isn't tied to a juju version, it should always just be bundle
> singular.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/charm-tools/pull/185/files/e28b4512527c33b9ab978471c6add644ad125869#r60173042>
>
-- 
Adam Israel - adam@adamisrael.com
www <http://www.adamisrael.com/> :: Twitter
<http://www.twitter.com/adamisrael> :: Facebook
<https://www.facebook.com/adam.israel> :: Inkpunks
<http://www.inkpunks.com/>